src/VisualStudio/Core/Def/InheritanceMargin/MarginGlyph/MemberMenuItemViewModel.cs

```diff
@@ -35,26 +37,15 @@ internal class MemberMenuItemViewModel : MenuItemViewModel
         public MemberMenuItemViewModel(
             string displayContent,
             ImageMoniker imageMoniker,
-            string automationName,
-            ImmutableArray<MenuItemViewModel> targets) : base(displayContent, imageMoniker, automationName)
+            ImmutableArray<MenuItemViewModel> targets) : base(displayContent, imageMoniker)
         {
             Targets = targets;
         }
 
         public static MemberMenuItemViewModel CreateWithHeaderInTargets(InheritanceMarginItem member)
-        {
-            var displayName = member.DisplayTexts.JoinText();
-            var targetsByRelationship = member.TargetItems
-                .OrderBy(item => item.DisplayName)
-                .GroupBy(target => target.RelationToMember)
-                .SelectMany(grouping => InheritanceMarginHelpers.CreateMenuItemsWithHeader(grouping.Key, grouping))
-                .ToImmutableArray();
```

Comment:
this helper was redundant.  now we just use CreateModelsForTargetItems which already did this work.